auth-web/.env.example

```diff
@@ -1,22 +1,16 @@
-# Base Path Opensift: /business/auth/ Firebase: /
 VUE_APP_PATH="/"
 
-#vaults Shared
+# vaults canadapost
 VUE_APP_ADDRESS_COMPLETE_KEY=
 
-#vaults auth-web
+# vaults auth-web
 VUE_APP_AFFIDAVIT_FILE_SIZE="73KB"
 VUE_APP_PAYMENT_PAYEE_NAME="BC Registries"
 VUE_APP_FILE_SERVER_URL="https://storage.googleapis.com/auth-static-resources-dev"
 VUE_APP_APPROVE_ACCOUNT_SLA_DAYS="3-5"
 VUE_APP_ONE_STOP_URL="https://dev.onestop.gov.bc.ca/"
-VUE_APP_SOCIETIES_URL="https://www.bcregistry.ca/societies/"
-VUE_APP_CORP_FORMS_URL="https://www2.gov.bc.ca/gov/content/employment-business/business/managing-a-business/permits-licences/businesses-incorporated-companies/forms-corporate-registry"
-VUE_APP_LLP_FORMS_URL="https://www2.gov.bc.ca/assets/gov/employment-business-and-economic-development/business-management/permits-licences-and-registration/registries-packages/pack_01_llp_-_registration_forms_package.pdf"
-VUE_APP_LP_FORMS_URL="https://www2.gov.bc.ca/assets/gov/employment-business-and-economic-development/business-management/permits-licences-and-registration/registries-forms/reg_791_-_declaration_for_bc_limited_partnership.pdf"
-VUE_APP_XLP_FORMS_URL="https://www2.gov.bc.ca/assets/gov/employment-business-and-economic-development/business-management/permits-licences-and-registration/registries-forms/reg_790_-_declaration_for_extraprovincial_limited_partnership.pdf"
```

Comment:
These are actually in web-url vault so I moved them down so that the items in this file match the items in the vaults file.